subsys/caf/modules/power_manager.c

```diff
@@ -148,19 +150,32 @@ static void system_off_on_error(void)
 	LOG_WRN("System turned off because of unrecoverable error");
 	LOG_PANIC();
 
+#if !defined(CONFIG_SOC_SERIES_NRF54HX)
```

Comment:
1) Yes of course I should have add CONFIG_PM thanks for finding it
2) NRF54L seams to work fine with old power management and also lets concentrate on one task at the time so I wouldn't make changes there in this task. 
3) Completly don't understand your idea with pm_state_force and extra state and extra policy - why should we do it? For now we can't enter state soft_off so lets not assume how it will be done in the future will we enter it using pm api or some other function like sys_poweroff(). Now we don't enter it and let's just leave it as it is.

Reply:
> NRF54L seams to work fine with old power management and also lets concentrate on one task at the time so I wouldn't make changes there in this task.

Actually we have not confirmed lower power consumption for nRF54L yet (unless you did it already?). Please just double check with people involved in PM development if the `CONFIG_PM` will be beneficial for nRF54L or there is no need to enable it there and we can stick to `sys_poweroff` API. It would be good to make an informed decision here already.

Reply:
> Completly don't understand your idea with pm_state_force and extra state and extra policy - why should we do it? For now we can't enter state soft_off so lets not assume how it will be done in the future will we enter it using pm api or some other function like sys_poweroff(). Now we don't enter it and let's just leave it as it is.

From what I heard so far, I would expect we will use PM APIs here. Still, maybe you already got newer information that would state otherwise. I would not insist here.